src/lab.rs

```diff
@@ -259,21 +262,21 @@ mod test {
     #[test]
     fn red() {
         let a = Lab::from(Rgb::new(1.0, 0.0, 0.0));
-        let b = Lab::new(53.23288 / 100.0, 80.10933 / 128.0, 67.22006 / 128.0);
+        let b = Lab::new(53.23288 / 100.0, 80.09246 / 100.0, 67.2031 / 100.0);
         assert_approx_eq!(a, b, [l, a, b]);
     }
 
     #[test]
     fn green() {
         let a = Lab::from(Rgb::new(0.0, 1.0, 0.0));
-        let b = Lab::new(87.73704 / 100.0, -86.184654 / 128.0, 83.18117 / 128.0);
+        let b = Lab::new(87.73704 / 100.0, -86.184654 / 100.0, 83.18117 / 100.0);
         assert_approx_eq!(a, b, [l, a, b]);
     }
 
     #[test]
     fn blue() {
         let a = Lab::from(Rgb::new(0.0, 0.0, 1.0));
-        let b = Lab::new(32.302586 / 100.0, 79.19668 / 128.0, -107.863686 / 128.0);
+        let b = Lab::new(32.302586 / 100.0, 79.19668 / 100.0, -107.863686 / 100.0);
```

Comment:
`-107.863686` tells me that it can easily reach outside +-100. I would love it if this wasn't the case, but...